lib/components/BulletList/BulletList.tsx

```diff
@@ -1,10 +1,44 @@
-import React, { ReactNode } from 'react';
+import React, { ReactNode, createContext, Children } from 'react';
 import { Box } from '../Box/Box';
+import { UseTextProps } from '../../hooks/typography';
+import { SpaceY } from '../../hooks/useBox';
+
+const defaultSize = 'standard';
+const defaultSpace = 'xxsmall';
+interface BulletListContext {
+  size: UseTextProps['size'];
+  space?: SpaceY;
```

Comment:
Should this be a mandatory field? The context object should always have everything defined, right?

Reply:
I could go either way, i refactored this to be optional so that the spacing logic lived in `BulletList` not `Bullet`. `Bullet` is then naive and renders `space` from context unconditionally. I could pass `none`, but micro-optimised to `undefined`.